src/components/Textarea/Textarea.test.tsx

```diff
@@ -51,4 +51,40 @@ describe("Textarea ", () => {
     rerender(<Textarea value={testValue} />);
     expect(textarea).toHaveValue("");
   });
+
+  it("adds Ctrl + Enter keydown event listener on mount and removes on unmount", async () => {
+    const addEventListenerSpy = jest.spyOn(document, "addEventListener");
+    const removeEventListenerSpy = jest.spyOn(document, "removeEventListener");
+
+    const { unmount } = render(
+      <Textarea
+        onControlEnter={() => {
+          console.log("OnControlEnter was here.");
```

Comment:
Nit: please make phrasing more clear ("onControlEnter called/invoked")

Reply:
Nice idea, I've changed that locally and will push shortly.